packages/lesswrong/lib/notificationTypes.tsx

```diff
@@ -969,3 +933,82 @@ export const NewMentionNotification = registerNotificationType({
     );
   },
 })
+
+const notificationTypes: Record<string,NotificationType> = {
```

Comment:
This can be a `satisfies Record<string, NotificationType>` after the declaration if you want the type safety but also want to prevent type erasure.  (Otherwise it should probably be a `Partial<Record<...>>` to correctly return Maybe<> if you don't check whether the type name exists in the map.)